theano/gpuarray/dnn.py

```diff
@@ -1135,6 +1135,11 @@ def dnn_conv3d(img, kerns, border_mode='valid', subsample=(1, 1, 1),
                                     desc_op.subsample)
     out_shp = assert_conv_shape(out_shp)
     out = gpu_alloc_empty(ctx_name, dtype=img.dtype)(*out_shp)
+    if config.dnn.conv.algo_bwd_filter != 'none':
+        warnings.warn('[new backend] cuDNN backward filter convolution computation for 3D convolutions '
```

Comment:
Remove the "[new backend]" part of the message and idem in the old back-end. It don't help user.

Can you be more specific? As the error happen only for subsample on the axis that cause problems, can you check that?

Also, the check should be moved to the make_node of the op. Otherwise, user won't get the warning then they build the graph on the CPU and let the optimizer move it to the GPU.